examples/chat/README.md

```diff
@@ -0,0 +1,35 @@
+# Chat Application
+
+## Overview
+
+This application shows how to use the Ballerina WebSocket package to implement a simple chat application.
+
+## Implementation
+
+### 1. The WebSocket Server
+The WebSocket server is in charge of registering users to the chat application and broadcasting the messages received by the users.
+
+### 2. Users - WebSocket Clients
+Clients can register for the chat application by sending requests to the WebSocket server. When the application starts it will request a username. Once a non-empty username is entered, the user will get registered.
+
+After getting registered, the users can send messages to the chat group by typing messages on the console and pressing `Enter`. Then the server will broadcast messages by looping over the registered clients.
+If a user wants to exit from the chat, he/she can type in `exit` and `Enter`, so the connection will get closed, and the client gets unregistered from the chat.
```

Comment:
```suggestion
If a user wants to exit from the chat, he/she can type in `exit` and `Enter` so that the connection will get closed, and the client gets unregistered from the chat.
```